docs/sphinx/users/imagej/index.txt

```diff
@@ -1,7 +1,7 @@
 ImageJ overview
 ===============
 
-`ImageJ <http://rsb.info.nih.gov/ij/>`_ is an image processing and
+`ImageJ <https://imagej.nih.gov/ij/index.html>`_ is an image processing and
```

Comment:
Is http://imagej.net/Welcome the newer version of the ImageJ homepage? If so should we update it here